config/rbac/role.yaml

```diff
@@ -18,18 +18,6 @@ rules:
   - patch
   - update
   - watch
-- apiGroups:
-  - apps
-  resources:
-  - deployments
```

Comment:
I'm curious -- MCAD can still create deployments right? So won't we still need deployment perms for the CFO?

Reply:
Right. The diff makes it hard to see, but this was actually duplicated, and there are still permissions for Deployments a bit below.